### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by chrmod**

### CODE REVIEW

**Review by philipp-classen — APPROVED** [review]

File: src/manifest.json [hunk 1]
```diff
@@ -59,22 +63,25 @@
...
   ],
   "chrome_settings_overrides" : {
     "search_provider": {
-      "name": "Ghostery Search",
-      "search_url": "https://ghosterysearch.com/search?q={searchTerms}",
-      "keyword": "@ghostery",
-      "favicon_url": "https://ghosterysearch.com/img/favicon-32x32.png",
+      "name": "Ghostery Glow",
+      "search_url": "https://glowstery.com/search?q={searchTerms}",
+      "keyword": "@glowstery",
```
Comment (remusao) [inline src/manifest.json]: Should we really change this keyword? Is it used to trigger custom search from URL bar?

File: src/background.js [hunk 2]
```diff
@@ -111,11 +111,26 @@ async function start() {
     return {
       requestHeaders,
     };
-  }, { urls: [`${SERP_BASE_URL}/search*`]}, ["blocking", "requestHeaders"]);
+  }, { urls: [`${SERP_BASE_URL}/search*`, USE_STAGING ? 'https://staging.ghosterysearch.com/search*' : 'https://ghosterysearch.com/search*']}, ["blocking", "requestHeaders"]);
+
+  // TODO: this should only run in Ghostery Dawn
```
Comment (remusao) [inline src/background.js:116]: Is there already a ticket for this?

File: src/background.js [hunk 3]
```diff
@@ -111,11 +111,26 @@ async function start() {
     return {
       requestHeaders,
     };
-  }, { urls: [`${SERP_BASE_URL}/search*`]}, ["blocking", "requestHeaders"]);
+  }, { urls: [`${SERP_BASE_URL}/search*`, USE_STAGING ? 'https://staging.ghosterysearch.com/search*' : 'https://ghosterysearch.com/search*']}, ["blocking", "requestHeaders"]);
```
Comment (remusao) [inline src/background.js:114]: Should this be:
```suggestion
  }, { urls: [`${SERP_BASE_URL}/search*`, USE_STAGING ? 'https://staging.glowstery.com/search*' : 'https://glowstery.com/search*']}, ["blocking", "requestHeaders"]);
```

File: src/manifest.json (hunk 1 above)
Comment (chrmod) [reply src/manifest.json]: yeah, and only in the case Glow is not default. but yeah, valid point. I will rollback to Ghostery.

File: src/background.js (hunk 2 above)
Comment (chrmod) [reply src/background.js:116]: Nope.

File: src/background.js (hunk 3 above)
Comment (chrmod) [reply src/background.js:114]: No, `*.ghosterysearch.com` is from now considered deprecated and will be removed in next release. This is just for transition period.

**Pull request merged**